about.html

```diff
@@ -197,7 +197,66 @@
               <h1 style="font-weight: 300">Introducing Aqovia</h1>
               <p style="color: #FFFFFF; max-width: 90%; width: 800px"> Digital solutions that empower businesses and human talent and offer new pathways to harness change - that is what we are all about. Our business centric value approach ensures low risk and high reward. </p>
           <p style="color: #FFFFFF">
-			 <div> <img src="img/scroll-anim.gif" width="25px"> </div></div>
+			  <div align="center">
+	 <svg class="arrows">
+							<path class="a1" d="M0 0 L30 32 L60 0"></path>
+							<path class="a2" d="M0 20 L30 52 L60 20"></path>
+							<path class="a3" d="M0 40 L30 72 L60 40"></path>
+						</svg>
+	</div>
+	 
+	 <style>
+	
+
+.arrows {
+	width: 60px;
+	height: 72px;
+	position: ;
+	left: 50%;
+	margin-left: -30px;
+	margin-top: 50px;
+	bottom: 20px;
+}
+
+.arrows path {
+	stroke: #fff;
+	fill: transparent;
+	stroke-width: 1px;	
+	animation: arrow 2s infinite;
+	-webkit-animation: arrow 2s infinite; 
+}
+
+@keyframes arrow
+{
+0% {opacity:0}
+40% {opacity:1}
+80% {opacity:0}
+100% {opacity:0}
+}
+
+@-webkit-keyframes arrow /*Safari and Chrome*/
+{
+0% {opacity:0}
+40% {opacity:1}
+80% {opacity:0}
+100% {opacity:0}
+}
+
+.arrows path.a1 {
+	animation-delay:-1s;
+	-webkit-animation-delay:-1s; /* Safari 和 Chrome */
+}
+
+.arrows path.a2 {
+	animation-delay:-0.5s;
+	-webkit-animation-delay:-0.5s; /* Safari 和 Chrome */
+}
+
+.arrows path.a3 {	
+	animation-delay:0s;
+	-webkit-animation-delay:0s; /* Safari 和 Chrome */
+}
+	 </style></div>

```

Comment:
I feel it would be better to have styles in their own stylesheets or to the main stylesheets as peppering the HTML with bits of CSS would make it much harder to maintain and understant